### ISSUE TITLE
Simplify recording a command

### ISSUE THREAD

**Issue by sindresorhus** [marionebl/svg-term-cli#7]
My most common use-case for this CLI is to record a command with `asciinema` and pass it to `svg-term-cli` without saving the recording. Would be nice if that was as simple as using a flag here.

Currently, I have to do:

```
temp_file=$(mktemp); asciinema rec $temp_file --command="node example.js" --quiet; cat $temp_file | svg-term --out=screenshot.svg
``` 

Would be nicer to be able to do:

```
svg-term --command="node example.js" --out=screenshot.png
```

`svg-term-cli` would then just spawn `asciinema` or tell the user to `brew install asciinema` if it's not installed.

**Comment by marionebl** [comment]
May become easier when https://github.com/asciinema/asciinema/issues/255 gets resolved

**Comment by marionebl** [comment]
The prerelease versions `2.1.0-x` provide a `--command` flag as proposed, progress is tracked here: https://github.com/marionebl/svg-term-cli/pull/8

**Comment by marionebl** [comment]
Released on `@latest` as of `2.1.0`
